tests/Browser/Pagination/Test.php

```diff
@@ -137,7 +243,7 @@ public function it_can_have_two_sets_of_links_for_the_one_paginator_on_a_page()
     public function it_calls_pagination_hook_method_when_pagination_changes()
     {
         $this->browse(function ($browser) {
-            Livewire::visit($browser, ComponentWithPaginationHook::class)
+            Livewire::visit($browser, ComponentWithCursorPaginationBootstrap::class)
```

Comment:
@ariaieboy can you confirm why this test was changed? I think you should probably leave it as it was and add another test for cursor pagination hooks if it's needed 🙂

Reply:
phpstorm changed it i've change it back to ComponentWithPaginationHook

Reply:
Ok great thanks!